cves/CVE-2014-9604.yml

```diff
@@ -129,7 +130,7 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: This vulnerability was left unfixed and unnoticed for over 2 years and over 2 major version updates of FFmpeg
```

Comment:
Wow, thats quite a while. Were there any changes to the code specific to or around this vulnerability during that time?